fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -124,10 +124,10 @@ public class DatabaseTransactionMgr {
 
 
     // count the number of running txns of database, except for the routine load txn
-    private int runningTxnNums = 0;
+    private volatile int runningTxnNums = 0;
 
     // count only the number of running routine load txns of database
-    private int runningRoutineLoadTxnNums = 0;
+    private volatile int runningRoutineLoadTxnNums = 0;
```

Comment:
Do we meet errors using int? If there are any errors, I think you'd better add comment here.